src/commands/schemas.ts

```diff
@@ -357,26 +352,11 @@ async function deleteSchemaVersionCommand(schema: Schema) {
       schemaVersionsChanged.fire({ change: "deleted", subject: newSubject });
     }
   } catch (e) {
-    success = false;
     logError(e, "Error deleting schema version", { extra: { error: {} } });
     showErrorNotificationWithButtons(
       `Error deleting schema version ${schema.version}: ${e instanceof Error ? e.message : String(e)}`,
     );
   }
-
-  logUsage(UserEvent.SchemaAction, {
```

Comment:
Moving these away from here then lose what is being provided in the `status` column, breaking continuity with existing Segment events and the column in our BigQuery / SuperSet table.